client/src/Auth/authContainer.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react';
+import Register from './Register/register.js';
+import axios from 'axios';
+
+const defaultState = {
+    username: '',
+    password: '',
+    confirmPassword: ''
+};
+
+class Auth extends Component {
+    constructor (props) {
+        super();
+         
+        this.state = defaultState;
+    }
+
+    clearState = () => {
+        this.setState(defaultState);
+    }
+
+    handleChange = (input, e) => {
+        this.setState({
+            [input]: e.target.value
+        })
+    }
+
+    handleSubmit = () => {
+
+        axios.post('/api/register')
```

Comment:
How are you passing the data to the api?

Reply:
Since, we didn't have the registration up I left the object blank, but you are right, i'll place it in the api call.